src/Compilers/CSharp/Portable/SymbolDisplay/ObjectDisplay.cs

```diff
@@ -133,115 +133,115 @@ internal static string FormatLiteral(bool value)
             return value ? "true" : "false";
         }
 
-        internal static string FormatString(string str, char quote, bool escapeNonPrintable)
+        private static void FormatChar(
```

Comment:
Personally, I might have called this FormatStringChar to indicate that it is not a general method, but rather a helper method for formatting a single character in a longer string.